sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/MetricExporterConfiguration.java

```diff
@@ -57,6 +57,11 @@ static void configureExporter(
         metricExporter = spiExporter;
     }
 
+    // exporter may be null in otlp case
```

Comment:
Any idea why null away didn't catch this? Guess it must allow null into java Function :(

Reply:
> Any idea why null away didn't catch this

I was curious about that as well. There's a related [discussion](https://github.com/uber/NullAway/issues/387#issuecomment-609895020) that explains that `Function` is maximally permissive, with `@Nullable` parameters and `@NonNull` response.